### ISSUE TITLE
python 3.6?

### ISSUE THREAD

**Issue by Korijn** [harvimt/quamash#72]
Any efforts going on to support python 3.6? Are you accepting PRs?

**Comment by harvimt** [comment]
Is that a question? This package only works with Python 3.4 or greater.

**Comment by Korijn** [comment]
The README doesn't state that 3.6 is supported by this library. So I decided to ask if it is the case.

If it is merely outdated information I can open a PR to correct it for you?

**Comment by harvimt** [comment]
ugh. this is one of those times where having a working test suite would be good, because then we could re-run the test suite on 3.6 and then we'd know it works.

It should work just fine on 3.6 though.

**Comment by Korijn** [comment]
Fixing Travis seems simple enough; just two flake8 errors.

```
./quamash/_unix.py:220:1: E305 expected 2 blank lines after class or function definition, found 1
./quamash/__init__.py:172:1: E305 expected 2 blank lines after class or function definition, found 1
```
From https://travis-ci.org/harvimt/quamash/jobs/188431070

Is there a big issue holding you back that I'm overlooking? If it's just some linting I'll fix it for you.

**Comment by Korijn** [comment]
Lots of errors on AppVeyor I see: https://ci.appveyor.com/project/harvimt/quamash

Is that the big issue?

**Comment by harvimt** [comment]
Last I checked Travis was failing to install PyQt, but I guess hadn't
looked in a while?

The Appveyor tests are another thing, getting all the tests to pass on
windows would be good (but not necessary to declare compat with python 3.6)

**Comment by Korijn** [comment]
I opened PR #74 to get CI to pass first.

Once that is merged I will open another PR to enable Python 3.6.
